content/en/opentelemetry/agent/agent_with_custom_components.md

```diff
@@ -0,0 +1,258 @@
+# You will see a list of extensions already included by Datadog 
+# Add your desired extensions here
+
+exporters:
+# You will see a list of exporters already included by Datadog 
+# Add your desired exporters here
+
+processors:
+# You will see a list of processors already included by Datadog 
+# Add your desired processors here
+
+receivers:
+  - gomod: go.opentelemetry.io/collector/receiver/nopreceiver v0.104.0
+  - gomod: go.opentelemetry.io/collector/receiver/otlpreceiver v0.104.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/filelogreceiver v0.104.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/fluentforwardreceiver v0.104.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver v0.104.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jaegerreceiver v0.104.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver v0.104.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/receivercreator v0.104.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinreceiver v0.104.0
+# adding Kafka metrics receiver to collect Kafka metrics in OTLP format
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkametricsreceiver v0.104.0
+
+connectors:
+# You will see a list of connectors already included by Datadog 
+# Add your desired connectors here
+{{< /highlight >}}
+1. Save your changes to the manifest file. 
+
+## Build and push the Agent image
+
+Build your custom Datadog Agent image and push it to a container registry.
+
+1. Build the image with Docker:
+   ```shell
+   docker build . -t agent-otel --no-cache
+   ```
+2. Tag and push the image:
+   ```shell
+   docker tag agent-otel <IMAGE-NAME>/<IMAGE-TAG>
+   docker push <IMAGE-NAME>/<IMAGE-TAG>
+   ``` 
+   Replace `<IMAGE-NAME>` and `<IMAGE-TAG>` with your image name and desired tag. If the target repository is not Docker Hub, you need to include the repository name. 
+3. For a Helm Chart installation, set the image tag in your values file:
+   {{< code-block lang="yaml" filename="datadog-values.yaml" collapsible="true" >}}
+agents:
+  image:
+    repository: <YOUR-REPO>
+    tag: <IMAGE-TAG>
+    doNotCheckTag: true
+   {{< /code-block >}}
+   Replace `<YOUR-REPO>` and `<IMAGE-TAG>` with your repository name and desired image tag.
+   
+<div class="alert alert-info">For complete installation instructions, see <a href="/opentelemetry/agent/install_agent_with_collector">Install the Datadog Agent with Embedded OpenTelemetry Collector</a>.</div>
+
+## Test and validate
+
+Create a sample configuration file and run your custom Agent to ensure everything is working correctly.
+
+1. Create a sample OTel configuration file with the additional components.  
+   Here is an example with the additional [Kafka metrics receiver][7] configured:
```

Comment:
```suggestion
  The following example configures an additional [Kafka metrics receiver][7]:
```

Optional suggestion— I just feel like "here is" doesn't seem right with our style? But this is really just a feeling rather than a rule